docs/src/validator/best-practices/operations.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Validator Operations Best Practices
+sidebar_label: General Operations
+---
+
+After you have successfully gotten a [validator running on testnet](../get-started/setup-a-validator.md) (or another cluster of your choice), you will want to become familiar with how to operate your validator on a day-to-day basis. During daily operations, you will be [monitoring your server](./monitoring.md), updating software regularly (both the Solana validator software and operating system packages), and managing your stake account and identity account.
```

Comment:
suggest replace "gotten" with "setup" or "got", "gotten" is a North American english not commonly used elsewhere in the world. 

Replace "stake account" with "vote account" in the last line of this paragraph

Reply:
Thank you. Updated.